### DISCUSSION TITLE
Culture is not supported (Parameter 'name') us-EN is an invalid culture identifier

### DISCUSSION THREAD

**Discussion by mirzakarim** [porrey/Virtual-ZPL-Printer#64] [Q&A]
Used on 2 machines. On one machine everything is fine. on another machine get this error message on any attemp to send a document (including Test) 
![image](https://github.com/user-attachments/assets/2f10d653-5ed4-4798-a31a-f5f2d8e743e2)
Changed the locale (Sweden -> EN US, then to EN GB). Didn't help. What else can be done

**Comment by porrey** [comment]
When making the call to the Labelary API, a conversion to Enlglish is performed because the API only supports English. You may need to load English in your language settings.

<img width="1025" alt="Screenshot 2024-07-12 at 2 59 57 PM" src="https://github.com/user-attachments/assets/ad1352d1-3b72-490c-9d25-13cddb7645bf">

If you already have English, you may have some issue with your Windows setup.
